CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+Hi, we're glad that you're interested in contributing to **kiwee/shopware-storefront-build-config** !
+
+Please take a moment to read these guidelines.
+
+## Submitting issues
+
+* Check is a similar issue already exists (it may be closed)
+* When reporting bugs please include all information that may help us reproduce the issue. Make sure to include at least **the context, steps to reproduce, what were you expecting to happen, and what happened instead**
+* Create and share a minimal, reproducible example to get the issue fixed quicker
+
+## Submitting pull requests
+
+* For any larger changes or new features create an issue first
+* For small bug fixes you can skip the issue and only create a pull request
+* Make sure to write [clear commit messages](https://cbea.ms/git-commit/) for all your commits
+
+## Support
+
+If you have any questions about this project please start or join a [discussion](https://github.com/KiweeEu/magento2-on-kubernetes/discussions).
```

Comment:
the link should not target MOK I believe.

Reply:
Definitely. Good catch, thanks.